modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java

```diff
@@ -149,11 +146,13 @@ public RustClientCodegen() {
         typeMapping.clear();
         typeMapping.put("integer", "i32");
         typeMapping.put("long", "i64");
-        typeMapping.put("number", "f32");
+        typeMapping.put("number", "f64");
```

Comment:
It was wrong - number should be `f64`. 
And currently master branch already have breaking changes unfortunately - see #17869

This can be reverted though, but I thought it's nice to get fixed

Reply:
Sounds good, we should just mention it somewhere.

Reply:
Added to the description of the PR. As well as mention of import changes